CHANGELOG.md

```diff
@@ -344,6 +360,10 @@ List of new modules
 
   Axiom.UIP
   Axiom.UIP.WithK
+  Axiom.ExcludedMiddle
+  Axiom.DoubleNegationElimination
+  Axiom.Extensionality.Propositional
+  Axiom.Extensionality.Heterogeneous
 
```

Comment:
I think we should either expand `UIP` to `UniquenessOfIdentityProof` or adopt
the same naming style for `EM`, `DNE` and `FunExt`.

Reply:
Note to self, forgot to change CHANGELOG entry.